src/index.js

```diff
@@ -561,7 +563,7 @@ export default class Carousel extends React.Component {
           this.previousSlide();
         }
       }
-    } else {
+    } else if (touchLength > 0) {
```

Comment:
Super small fix, just making sure when a touch happens, it has some distance before we trigger the `goToSlide` to slide back to the `currentSlide`.